composer.json

```
@@ -16,9 +16,12 @@
             "homepage": "http://mouf-php.com"
         }
     ],
+    "scripts": {
+        "test": "phpunit"
+    },
     "require": {
-        "php": ">=7.2",
-        "thecodingmachine/graphqlite": "^4",
+        "php": ">=7.2|8.0",
```

No need to specify 8.0 version, since ">=7.2" covers it.

```suggestion
        "php": ">=7.2",
```